src/Functions/array/arrayElement.cpp

```diff
@@ -18,6 +18,8 @@
 #include <Columns/ColumnMap.h>
 #include <Common/typeid_cast.h>
 #include <Common/assert_cast.h>
+#include <Poco/Logger.h>
+#include <Common/logger_useful.h>
```

Comment:
```suggestion
```

Ops, I somehow messed up my review and I haven't commented about this. I think this is not necessary as we don't log anything here.